draft-ietf-webtrans-http3.md

```diff
@@ -974,11 +1005,13 @@ different trust domains, meaning that the implementations have to treat each
 transport as potentially hostile towards others on the same connection.  One
 potential attack is a resource exhaustion attack: since all of the WebTransport
 sessions share both congestion control and flow control context, a single
-session aggressively using up those resources can cause other sessions to
-stall.  A WebTransport endpoint thus SHOULD implement a fairness scheme that
-ensures that each session that shares a transport connection gets a reasonable
-share of controlled resources; this applies both to sending data and to opening
-new streams.
+application aggressively using up those resources can cause other sessions to
+stall.  A WebTransport endpoint MUST implement flow control mechanisms if it
+allows a WebTransport session to share the transport connection with other
+WebTransport sessions.  WebTransport endpoints SHOULD implement a fairness
```

Comment:
Do we also need to mention sharing WebTransport sessions with HTTP requests?

Reply:
I started by saying that you have to do flow control if you share with HTTP requests, but we don't really enforce that via the setting in the same way, so took that out for today. But yeah, I think it would be nice to say if you share with HTTP requests, you have to do flow control as well to protect those streams -- it allows the collection of streams that form a WT session to not starve the other ones.